src/lib/custom-icons/index.ts

```diff
@@ -0,0 +1,86 @@
+import {
```

Comment:
@poojahpatil90 , I like this approach, thanks 🙏 💯

Reply:
Thanks 🙏